ch.protonmail.protonmail-bridge.appdata.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+  <id>ch.protonmail.protonmail-bridge</id>
+  <name>Protonmail Bridge</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <translation/>
+  <developer_name>Proton Technologies AG</developer_name>
+
+  <summary>
+    Seamlessly encrypts and decrypts your mail as it enters and leaves your computer
+  </summary>
+
+  <url type="homepage">
+    https://protonmail.ch/bridge/
+  </url>
+
+  <launchable type="desktop-id">
+    ch.protonmail.protonmail-bridge.desktop
+  </launchable>
+
+  <description>
+    <p>
+      The ProtonMail Bridge is an application for paid users that runs on your computer in the background and seamlessly encrypts and decrypts your mail as it enters and leaves your computer. It allows for full integration of your ProtonMail account with any program that supports IMAP and SMTP such as Microsoft Outlook, Mozilla Thunderbird and Apple Mail.
+    </p>
+  </description>
+
+  <screenshots>
+
+    <screenshot>
+      <image type="source">
+	https://raw.githubusercontent.com/A6GibKm/share/master/protonmail-bridge/bridge.png
+      </image>
+    </screenshot>
+
+    <screenshot>
+      <image type="source">
+	https://raw.githubusercontent.com/A6GibKm/share/master/protonmail-bridge/bridge-conf.png
+      </image>
+    </screenshot>
+
+  </screenshots>
+
+  <releases>
+    <release version="1.1.6" date="2019-06-28">
+
+      <description>
+	<p>
+	  Information about v1.1.6 beta:
+	</p>
+	<ul>
+	  <li>Necessary updates reflecting API changes</li>
+	  <li>Report wrongly formated messages</li>
+	  <li>Fixed verification for contacts signed by older or missing key</li>
+	  <li>Outlook always shows attachment icon</li>
+	</ul>
+      </description>
+    </release>
+  </releases>
+
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">moderate</content_attribute>
+    <content_attribute id="social-audio">moderate</content_attribute>
+  </content_rating>
+
+  <update_contact>
+    bridge@protonmail.ch
```

Comment:
They didn't write this file so this is the wrong contact.

Reply:
This is the mail of the devs of the app.

Reply:
update_content is about the appdata file and has nothing to do with the software. Unless you convince upstream to ship this file of course.